packages/data/lib/utils/const.dart

```diff
@@ -12,4 +12,5 @@ class Config {
   static const peoplesEndpoint = '/people';
   static const tmdbBasePath = 'https://api.themoviedb.org';
   static const tmdbPersonPath = '/3/person';
+  static const userFirestoreCollection = 'users';
```

Comment:
need to split this class into several. for example, only timeout/apikeys can be related to config

paths and base url should relate to another class - ApiPath for ex

Firestore also can be separated